samples/react-contoso/src/pages/ChatPage.tsx

```diff
@@ -143,7 +144,7 @@ const ChatPage: React.FunctionComponent = () => {
           </Dialog>
         </div>
         <div className={styles.side}>
-          <ChatListWrapper selectedChatId={chatId} onSelected={onChatSelected} onNewChat={onNewChat} />
+          {isSignedIn && <ChatListWrapper selectedChatId={chatId} onSelected={onChatSelected} onNewChat={onNewChat} />}
```

Comment:
was this the intended implementation? The user would somehow need to know to conditionally render this based on isSignedIn. If it was a prop, we can do this conditional render or just short circuit within the component. Plus the user knows they must provide it

Reply:
This is a common approach, LGTM